README.md

```diff
@@ -10,6 +10,8 @@ We explore instruction-tuning popular base models on publicly available datasets
 
 As part of this work we introduce Tülu, a suite of LLaMa models fully-finetuned on a strong mix of datasets!
 
+The license we release these models under (along with the base model licenses) can be found in `licenses/license.txt` - just replace `<MODELNAME>` with the actual model name (i.e., the name on HuggingFace).
```

Comment:
We also have a `LICENSE` file in the root folder for the codebase. I am thinking we can rename this folder to `model_licenses` and rename `license.txt` as `tulu_license.txt`.

Reply:
Also, maybe have a separate section in the README for "Licensing"?

Reply:
Fair points - I've edited the PR to do both.